hardware_interface/include/hardware_interface/types/hardware_interface_type_values.hpp

```diff
@@ -15,6 +15,8 @@
 #ifndef HARDWARE_INTERFACE__TYPES__HARDWARE_INTERFACE_TYPE_VALUES_HPP_
 #define HARDWARE_INTERFACE__TYPES__HARDWARE_INTERFACE_TYPE_VALUES_HPP_
 
+#include <vector>
+
 namespace hardware_interface
 {
 /// Constant defining position interface
```

Comment:
I would actually just enhance them – but keep the comments because those are doxygen comment that are shown nicely in documentation.

```suggestion
/// Constant defining position interface name
```